website/content/docs/concepts/connection-workflows/multi-hop.mdx

```diff
@@ -14,73 +14,81 @@ inbound traffic to route through multiple network enclaves to reach the target s
 
-In multi-hop scenarios, there are typically three types of workers:
-1. **Ingress worker** - An ingress worker is a worker that is accessible by the client. The client initiates the connection to the ingress worker.
-1. **Intermediary worker** - An optional intermediary worker sits between ingress and egress workers as part of a multi-hop chain. There can be multiple intermediary workers as part of a multi-hop chain.
-1. **Egress worker** - An egress worker is a worker that can access the target. The egress worker initiates reverse proxy connections to intermediary or ingress workers.
+Multi-hop sessions let you connect to private endpoints without necessarily opening inbound access to isolated or private networks.
+By allowing outbound access from an egress worker in the private network to an upstream worker on a specific port, a Boundary client can access the target using a secure tunnel that is initiated by an ingress worker in the chain.
 
-<Tip>
-“Ingress,” “intermediary,” and “egress” are general ways to describe how the respective worker interfaces with resources, and a worker can act as more than one of those
-at a time. For example in the diagram below, the intermediary worker is also an egress worker since it can access a target.
-</Tip>
+## Introduction
 
-![Multi-hop session example showing ingress, intermediary, and egress workers](/img/concepts-multihop.png)
+Multi-hop sessions introduces five concepts for how workers function.
+You can categorize them from the point of view of a client and a worker.
 
-After the persistent connection chain is established between the workers, when you attempt to connect to a target host, you are automatically proxied from:
-1. Boundary client to ingress worker
-1. Ingress worker to intermediary worker, where applicable
-1. Ingress worker to egress worker
-1. Egress worker to desired target
+From a worker perspective, the following concepts apply:
+
+- Upstream worker: A worker that is closer in hop count to a controller.
+Upstream workers receive join requests from downstream workers.
+- Downstream worker: A worker that is closer in hop count to a target.
+Downstream workers initiate connections to upstream workers.
+
+From a client perspective, the following concepts apply:
+
+- Ingress worker: A worker that Boundary clients connect to.
+It is the first worker in the multi-hop chain.
+- Intermediate worker: A worker with upstream and downstream workers in the chain.
+These workers provide the fabric for multi-hop from ingress worker to egress worker.
+- Egress worker: The last worker int he multi-hop chain.
+Boundary client connections egress the proxy chain at this worker to reach the target.
+
+Note that ingress, intermediary, and egress are general ways to describe how the respective worker interfaces with resources.
+A worker can act as more than one of those at a time.
 
 ## Multi-hop worker capabilities
```

Comment:
Having read the preview, I'm thinking we move this section to be under the `Introduction` section, and we move the content from the `Introduction` section here and rename the header from `Multi-hop worker capabilities` to `Multi-hop worker concepts`.  We talk about things upstream and downstream here currently, and if we move that to be under the introduction, then that is a good segway to better explain the concepts so users understand what was just mentioned in more detail.

Reply:
> Having read the preview, I'm thinking we move this section to be under the `Introduction` section, and we move the content from the `Introduction` section here and rename the header from `Multi-hop worker capabilities` to `Multi-hop worker concepts`. We talk about things upstream and downstream here currently, and if we move that to be under the introduction, then that is a good segway to better explain the concepts so users understand what was just mentioned in more detail.

Yep, great suggestion! I just moved these around and I think it flows nicely. I think this topic is much improved :tada:

I am wondering if we should expand on why people would want to configure multi-hop? We briefly address the benefits in the first para, but I'm wondering if it could be stronger.

Reply:
I reread the intro and thought about it some more, and honestly, Im coming up without any improvements.  The value of multi-hop workers seems clear to me with that intro, but I also have been working with multi-hop workers a lot so perhaps another set of eyes on it would be good?

Reply:
Sounds good! I would like to bring in some other reviewers. Thank you for all your help @sdoyle88 . Robin and I still need to review your updates to the worker procedure too. We'll do those things and get back to you.